projects/js-packages/ai-client/src/hooks/use-save-to-media-library.ts

```diff
@@ -67,7 +70,11 @@ export default function useSaveToMediaLibrary() {
 
 									if ( image ) {
 										debug( 'Image uploaded to media library', image );
-										resolve( image );
+										resolve( {
+											id: image.id,
+											url: image.url,
+											mime_type: image.mime,
+										} );
```

Comment:
We could aim to use the property name in our code unified with one from wordpress core. I proposed mime_type cause I thought this is how the Attachment type looks in core - but probably got confused.

What do you think?